dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/TrackerTrackedEntitiesExportControllerTest.java

```diff
@@ -191,6 +193,31 @@ void getTrackedEntityById() {
     assertHasNoMember(json, "programOwners");
   }
 
+  @Test
+  void shouldGetTrackedEntitiesDisregardingSearchScope() {
```

Comment:
should we add `whenCaptureScopeUsed` at the end? 
that's what this is testing, right?